.travis.yml

```diff
@@ -1,42 +1,32 @@
 language: python
 sudo: false
-cache:
-- pip
+dist: trusty
+cache: pip
 python:
-- '2.7'
-- '3.4'
-- '3.5'
-- pypy
```

Comment:
That's good for PyPy, but Django officially doesn't support PyPy. Furthermore PyPy currently only implements CPython 2.7 syntax. Django will drop Python 2 support in Django 2.